CONTRIBUTING.md

```diff
@@ -24,6 +24,18 @@ npx turbo run dev --filter example-basic...
 
 Finally, visit <http://localhost:3000/swingset> in your browser. Any changes to the core `swingset` package will be rebuilt and reflect in the running example app.
 
+## Debugging the example app
+
+If you are iterating on the loader code, it might be helpful to use the Node debugger instead of relying on `console.log()` statements. To make this easier, we have included a VSCode launch configuration.
+
+First, run the `dev` task for `example-basic`'s dependencies:
+
+```shell-session
+npx turbo run dev --filter example-basic^...
+```
+
+Next, open the VSCode debug panel (`Ctrl+Shift+D` on Windows/Linux, `⇧+⌘+D` on macOS) and run the `Debug examples/basic app` launch configuration. This will start Next's development process and attach a debugger. Any `debugger;` statements you have added in Swingset's core should be triggered.
```

Comment:
```suggestion
Next, open the VSCode debug panel (`Ctrl+Shift+D` on Windows/Linux, `⇧+⌘+D` on macOS) and run the `Debug examples/basic app` launch configuration. This will start Next's development process and attach a debugger. Any [`debugger;`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger) statements you have added in Swingset's core should be triggered.
```